api-wrapper/services/api-wrapper/src/test/java/net/catenax/edc/apiwrapper/security/BasicAuthenticationServiceTest.java

```diff
@@ -1,59 +1,80 @@
 package net.catenax.edc.apiwrapper.security;
 
+import net.catenax.edc.apiwrapper.config.BasicAuthVaultLabels;
 import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
+import org.eclipse.dataspaceconnector.spi.result.Result;
+import org.eclipse.dataspaceconnector.spi.security.Vault;
 import org.junit.jupiter.api.Test;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.any;
+
+import org.powermock.reflect.Whitebox;
 
 public class BasicAuthenticationServiceTest {
 
+    //Correct with new things
+
     private final Monitor monitor = mock(Monitor.class);
-    private final Map<String, String> users = Map.of("hello", "myPassword");
+    private final Vault vault = mock(Vault.class);
+    private final List<BasicAuthVaultLabels> config = List.of(
+            new BasicAuthVaultLabels("usera", "api-basic-auth-usera"),
+            new BasicAuthVaultLabels("userb", "api-basic-auth-userb")
+    );
 
-    private final BasicAuthenticationService authenticationServiceTest = new BasicAuthenticationService(monitor, users);
+    private final BasicAuthenticationService authenticationServiceTest = new BasicAuthenticationService(monitor, vault, config);
 
     @Test
-    void isAuthenticated_shouldReturnNoAuthenticationHeaderSpecified() {
-
-        Map<String, List<String>> map = Map.of("Authorization", new ArrayList<>());
-        assertThat(authenticationServiceTest.isAuthenticated(map)).isEqualTo(false);
+    void CredentialsHaveNoAuthorizationLabel(){
+        Map<String, List<String>> mapCorrectPassword = Map.of("user", new ArrayList<>(Arrays.asList("user dXNlcjpwYXNzd29yZA==", "blablabla")));
```

Comment:
You can do this also on the other functions
```suggestion
        Map<String, List<String>> mapCorrectPassword = Map.of("user", List.of("user dXNlcjpwYXNzd29yZA==", "blablabla")));
```